PCbuild/pythoncore.vcxproj

```diff
@@ -675,6 +677,14 @@
                       Overwrite="true"
                       Condition="$(PyConfigHText.TrimEnd()) != $(OldPyConfigH.TrimEnd())" />
   </Target>
+  <Target Name="_BuildJIT" BeforeTargets="PrepareForBuild" Condition="'$(UseJIT)' == 'true'" DependsOnTargets="_UpdatePyconfig;FindPythonForBuild">
+    <PropertyGroup>
+      <JITArgs Condition="$(Platform) == 'Win32'">i686-pc-windows-msvc</JITArgs>
+      <JITArgs Condition="$(Platform) == 'x64'">x86_64-pc-windows-msvc</JITArgs>
+      <JITArgs Condition="$(Configuration) == 'Debug'">$(JITArgs) --debug</JITArgs>
+    </PropertyGroup>
+    <Exec Command='$(PythonForBuild) "$(PySourcePath)Tools\jit\build.py" $(JITArgs)' WorkingDirectory="$(IntDir)"/>
+  </Target>
```

Comment:
I'll try to figure it out. :)

Reply:
I wonder if we should just ignore the incremental build stuff here. The JIT build script itself exits immediately if the inputs and output are up-to-date (technically it treats everything in `Tools/jit/**/*` as an input, which seems like it would be tricky or burdensome to do here).

So it's not like it would save us any time, really. I could re-encode the same dependencies here, but then it would just be pretty redundant for no real gain.

Reply:
I don't know vcxproj, but does a target (JIT build) that gets run but doesn't recreate its outputs get treated as a no-op? e.g. as [Ninja](https://ninja-build.org/manual.html) describes it:

> restat
> > if present, causes Ninja to re-stat the command’s outputs after execution of the command. Each output whose modification time the command did not change will be treated as though it had never needed to be built. This may cause the output’s reverse dependencies to be removed from the list of pending build actions.

If it gets treated as a no-op, then re-running the JIT build every time just costs the cost of running a python script and exiting. If it *doesn't* get treated as a no-op, then it might force a pointless re-link for targets that use its output.

Reply:
It's better to skip in the build system. You can use wildcard inputs (I believe there are already some examples in the file?).

I'm pretty sure the files will be reevaluated each time, so it shouldn't cause a pointless re-link (MSBuild only does target dependencies, not files/outputs).

But you'd be surprised how much launching Python actually costs. I'm regularly tempted to go in and fix up one of our other regen steps that always runs, because it's a really obvious few seconds that I know I don't need to wait for if I fix how it's specified!

Reply:
As I said, I'm happy to go in after this is merged and figure out the dependency specifications, so if you'd rather ignore it for now feel free.